playbook/package.json

```diff
@@ -98,24 +98,24 @@
     "node-sass": "^4.12.0",
     "prop-types": "^15.6.2",
     "rails-erb-loader": "^5.5.1",
-    "react": "16.8.6",
+    "react": "^16.8.6",
     "react-animate-height": "^2.0.23",
-    "react-dom": "16.8.6",
+    "react-dom": "^16.8.6",
     "react-dropzone": "^10.2.2",
     "react-highlight-words": "^0.16.0",
-    "react-modal": "3.12.1",
+    "react-modal": "^3.12.1",
     "react-popper": "^2.1.0",
-    "react-select": "3.0.8",
+    "react-select": "^3.0.8",
     "svg-url-loader": "^4.0.0",
     "terser-webpack-plugin": "^3.1.0",
-    "trix": "1.3.0",
+    "trix": "^1.3.0",
     "webpack-cli": "^3.3.12",
     "webpacker-react": "^0.3.2"
   },
   "peerDependencies": {
-    "react": "16.8.6",
-    "react-dom": "16.8.6",
-    "trix": "1.3.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "trix": "^1.3.0",
```

Comment:
We should be careful here as what happens here is we tell the transpiler *not* to include this version, but instead require that the consumer of the library provide the version specified in their application. We are fairly confident that much later versions of React would still be backwards compatible with the transpiled JavaScript, but this is definitely a place that needs much more testing. `react` and `react-dom` as you can imagine are the main libs of concern here.